src/NuGet.Core/NuGet.Packaging/PackageArchiveReader.cs

```diff
@@ -360,7 +360,20 @@ public override string GetContentHashForSignedPackage(CancellationToken token)
             {
                 return SignedPackageArchiveUtility.GetPackageContentHash(reader);
             }
+        }
+
+        public override string GetContentHashForPackage(CancellationToken token)
+        {
+            var contentHash = GetContentHashForSignedPackage(token);
+            if (contentHash == null)
+            {
+                // Return the zip stream position to the begining
```

Comment:
beginning

But you really don't need the comment.  The next statement is clear enough.